src/ocspdash/cli.py

```diff
@@ -53,15 +60,22 @@ def nuke(connection, yes):
 
 
 @main.command()
-@click.option('--connection', help=f'SQLAlchemy connection. Defaults to {OCSPDASH_DEFAULT_CONNECTION}')
+@click.option(
+    '--connection',
+    help=f'SQLAlchemy connection. Defaults to {OCSPDASH_DEFAULT_CONNECTION}',
+)
 @click.argument('location_name')
 def new_location(connection, location_name):
     """Register a new location."""
     m = Manager.from_args(connection=connection)
 
     invite_id, invite_validator = m.create_location(location_name)
 
-    click.echo(base64.urlsafe_b64encode(b''.join((invite_id, invite_validator))).decode("utf-8"))
```

Comment:
this line is too long (and has too many inner calls) so you should change it to have more temporary variables